modules/administration/pages/image-management.adoc

```diff
@@ -1,12 +1,11 @@
 [[image-management]]
 = Image Building and Management
 
-
-
 [[at.images.overview]]
 == Image Building Overview
 
 {productname} enables system administrators to build containers and OS Images and push the result in image stores.
+
```

Comment:
I'd remove this empty line to avoid too many 1-sentence-paragraphs.

Reply:
Agreed, thanks.